src/ConfigWrapper/WindowsRegistryConfigWrapper.cs

```diff
@@ -185,16 +222,26 @@ public void Set<T>(string key, T value, bool createKeyIfNeeded)
         /// <inheritdocs />
         public void Set<T>(string key, T value)
         {
```

Comment:
Should the Set<T> method also consider key as FullKey(key) as the Get<T> method ?

Reply:
yes.  It does.  It calls Set<T>(string key, T value, bool createKeyIfNeeded).  That one first prepends the root key.